addons/l10n_bg/data/tax_report.xml

```diff
@@ -320,9 +321,92 @@
                             <record id="l10n_bg_tax_report_60_tag" model="account.report.expression">
                                 <field name="label">balance</field>
                                 <field name="engine">aggregation</field>
-                                <field name="formula">BG_TR_20.balance - BG_TR_40.balance</field>
+                                <field name="formula">BG_TR_40.balance - BG_TR_20.balance</field>
+                                <field name="subformula">if_above(BGN(0))</field>
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_bg_tax_report_d" model="account.report.line">
+                <field name="name">Section D. VAT for deposition</field>
+                <field name="children_ids">
+                    <record id="l10n_bg_tax_report_70" model="account.report.line">
+                        <field name="name">[70] Import tax from Article 50, deducted in accordance with Art. 92, para. 1 VAT</field>
+                        <field name="code">BG_TR_70</field>
+                        <field name="expression_ids">
+                            <record id="l10n_bg_tax_report_refund" model="account.report.expression">
+                                <field name="label">refund</field>
+                                <field name="date_scope">from_beginning</field>
+                                <field name="engine">account_codes</field>
+                                <field name="formula">4531 + 4538 + 4532 + 4539</field>
```

Comment:
Really minor thing but can you order them? :smile: 
And 4534/453 is not supposed to be part of this, right? 
Like, it's on purpose that we don't just define `453` as formula

Reply:
> And 4534/453 is not supposed to be part of this, right?

Ya, it looks like they aren't.